assets/app/views/browse/build-config.html

```diff
@@ -164,6 +164,7 @@
               <span ng-if="!(build | annotation : 'buildNumber')">
                 <a ng-href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfig.metadata.name}}/{{build.metadata.name}}">{{build.metadata.name}}</a>
               </span>
+              <span ng-if="build.status.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{build.status.message}}"></span>
```

Comment:
Any reason to use popover vs tooltip? We changed them all to tooltips once for visual consistency (#2330), but I see it's what we used for deployment config messages and pod warnings. I should have noticed on earlier pull requests.